### DISCUSSION TITLE
Eagerly waiting for Enhanced table for OSD 2.2.1 version

### DISCUSSION THREAD

**Discussion by Sandy381** [fbaligand/kibana-enhanced-table#287] [General]
Hi @fbaligand ,

I must say Enhanced table is really useful feature in Kibana/ OS Dashboards.

Thank you :)

I am using Opensearch 2.2.1. 

Any idea when Enhanced table for OSD 2.2.1 will be released.

**Comment by fbaligand** [comment]
Hi @Sandy381,

First, nice to see that you appreciate Enhanced Table plugin :)

Then, even if Opensearch 2.2.1 is out, the last release of Opensearch Dashboards is still 2.2.0:
https://github.com/opensearch-project/OpenSearch-Dashboards/releases

**Comment by fbaligand** [comment]
@Sandy381,

For information, since then, OpenSearch Dashboards 2.3.0 is out, and Enhanced Table for OSD 2.3.0 is available here:
https://github.com/fbaligand/kibana-enhanced-table/releases/tag/v1.13.3

**Reply by Sandy381** [reply]
Thank you :)
